README.md

```diff
@@ -7,7 +7,15 @@ NOTE: 'cron-master' library expected and required as a co-dependency
 
 ### Execution
 
-`require('tke-cron-master-builder')(cronConfigDir, collection, logCleanUpOpts);`
+```js
+const cronBuilder = require('tke-cron-master-builder')(cronConfigDir, collection, logCleanUpOpts);
+
+// load all jobs in cronConfigDir
+cronBuilder.loadJobs();
+
+// get handlers for a job we need to start programmatically
+cron.getHandlers(someJobInstance);
```

Comment:
The readme should probably explain this a little more. Where does someJobInstance come from? What does getHandlers() return? What would one do with them?